src/main/java/com/hivemq/cli/commands/options/TlsOptions.java

```diff
@@ -138,85 +137,298 @@ private boolean useBuiltSslConfig() {
                 .build();
     }
 
-    private void setDefaultOptions() {
+    private @Nullable KeyManagerFactory buildKeyManagerFactory() throws Exception {
+        final KeyManagerFactory keyManagerFactory;
+
         final DefaultCLIProperties defaultCLIProperties =
                 Objects.requireNonNull(MqttCLIMain.MQTTCLI).defaultCLIProperties();
-
-        if (clientCertificateChain == null) {
+        final Path defaultKeystore = defaultCLIProperties.getKeystore();
+        final Path defaultClientCertificateChain = defaultCLIProperties.getClientCertificateChain();
+        final Path defaultClientPrivateKey = defaultCLIProperties.getClientPrivateKey();
+
+        //keystore
+        if (clientKeystore != null) {
+            keyManagerFactory = createKeyManagerFactoryFromKeystore(clientKeystore,
+                    clientKeystorePassword,
+                    clientKeystorePrivateKeyPassword);
+            //certificate and private key file
```

Comment:
I think you need to re-align the comment if it is attached to the lower if clause. It is weird if it is part of the code block 